IS213-G9-tokyobanana/carbon-marketplace-backend
# Scheduler

## Context



Updated function that republishes task when scheduler supervisor notifies that previous task has failed.

## Test
Can be tested with @Jaylin0312 Supervisor PR. 
1. start supervisor MS & scheduler
2. publish message to task.status
3. check that new task has been added into task.execute queue
